src/main.cpp

```diff
@@ -12,25 +12,25 @@ int main(int argc, char *argv[]) {
 	std::string filename;
 	driver drv;
 	if (argv[argc - 1] == std::string("-h")) {
-		Frontend::Helper(helpArgument);
-		return 0;
+		Frontend::PrintHelper();
+		return 1;
```

Comment:
You're not supposed to return 1 when you print help after getting asked for it. Try running `grep --help` for example.